mmv1/third_party/terraform/tests/resource_compute_disk_test.go.erb

```diff
@@ -409,6 +409,64 @@ func TestAccComputeDisk_pdHyperDiskProvisionedIopsLifeCycle(t *testing.T) {
 	})
 }
 
+func TestAccComputeDisk_pdHyperDiskProvisionedThroughputLifeCycle(t *testing.T) {
+	t.Parallel()
+
+	context_1 := map[string]interface{}{
```

Comment:
For values that are fixed between all of these, can we bake them in to the test config?

Reply:
Yep I can do that. the Idea was to make it easier if they needed to change but I can simplify